src/templates/_layouts/parts-kit.html

```diff
@@ -1,111 +1,9 @@
-                                    {{ svg('@viget/base/resources/icons/pk-component.svg') | attr({
-                                        class: 'parts-kit__nav-component',
-                                    }) }}
-                                    <h3>{{ title | title }}</h3>
-                                </div>
-                            </button>
-
-                            <ol
-                                class="parts-kit__nav-sub-items {% if not section.isActive %}collapsed{% endif %}"
-                                id="{{ id }}"
-                                data-togglee
-                            >
-                                {% for item in section.items %}
-                                    <li data-section-item
-                                        data-search-text="{{ title ~ ' ' ~ item.title }}"
-                                    >
-                                        <a
-                                            href="{{ url(item.url) }}"
-                                            {% if item.isActive %}class="active"{% endif %}
-                                        >
-                                            <div class="flex">
-                                                {{ svg('@viget/base/resources/icons/pk-sidebar.svg') | attr({
-                                                    class: 'parts-kit__nav-story',
-                                                }) }}
-                                                {{ item.title }}
-                                            </div>
-                                        </a>
-                                    </li>
-                                {% endfor %}
-                            </ol>
-                        </li>
-                    {% endfor %}
-                </ol>
-            {% endif %}
-        </aside>
-
-        <div class="parts-kit__main">
-            <div class="parts-kit__main-wrapper">
-                <div class="parts-kit__main-header">
-                    <button
-                        class="parts-kit__button hidden hover:text-robins-egg"
-                        data-fullscreen-expand
-                    >
-                        <span class="parts-kit__sr-only">Expand</span>
-                        {{ svg('@viget/base/resources/icons/pk-expand.svg') }}
-                    </button>
-                    <button
-                        class="parts-kit__button hidden hover:text-robins-egg"
-                        data-fullscreen-close
-                    >
-                        <span class="parts-kit__sr-only">Close</span>
-                        {{ svg('@viget/base/resources/icons/close-circle.svg') }}
-                    </button>
-                </div>
-
-                <div class="parts-kit__main-container">
-                    {% block main %}{% endblock %}
-                </div>
-            </div>
-        </div>
-    </div>
+    {% block main %}
+    {% endblock %}
```

Comment:
Using the same template blocks as before.

The only difference is that for the root `/parts-kit` URL, we load a different layout that ONLY includes the JS for the parts kit. We don't want any styling or JS from the Craft site bleeding in to this page.